packages/eas-cli/src/commands/deploy/index.ts

```diff
@@ -32,6 +32,8 @@ import {
   getDeploymentUrlFromFullName,
 } from '../../worker/utils/logs';
 
+const MAX_UPLOAD_SIZE = 5e8; // 5MB
```

Comment:
`5e8` is 500MB (the comment is incorrect). Also could we name this sth like `MAX_TOTAL_UPLOAD_SIZE` or add a comment? At first glance it seemed like a per-asset limit.